src/components/oppsummering/vilkårsOppsummering/faktablokk/faktablokker/FormueFaktablokk.tsx

```diff
@@ -11,10 +11,10 @@ import { Nullable } from '~lib/types';
 import { Behandlingsinformasjon, FormueStatus } from '~types/Behandlingsinformasjon';
 import { SøknadInnhold } from '~types/Søknad';
 
-import Vilkårsblokk from '../../../vilkårsOppsummering/VilkårsBlokk';
-import saksbehandlingMessages from '../../formue/formue-nb';
-import { kalkulerFormue, kalkulerFormueFraSøknad } from '../../formue/utils';
-import { delerBoligMedFormatted } from '../../sharedUtils';
+import saksbehandlingMessages from '../../../../../pages/saksbehandling/steg/formue/formue-nb';
```

Comment:
Blir kanske nesten enklare å ha en relative path från root till pages istället for å gå bak så långt? 😊 f.eks `~/pages/saksbehandling/etc`, eller hvor nå pages ligger